tasks-by-users/AbbasShaikh0304/01-E-Commerce website/Style.css

```diff
@@ -0,0 +1,601 @@
+}
+
+/*-----------Single Product Details-----------*/
+
+.single-Product {
+  margin-top: 80px;
+}
+.single-Product .col-2 img {
+  padding: 0;
+}
+.single-Product .col-2 {
+  padding: 20px;
+}
+.single-Product h4 {
+  margin: 20px 0;
+
+  font-size: 22px;
+  font-weight: bold;
+}
+.single-Product select {
+  display: block;
+  padding: 10px;
+  margin-top: 20px;
+}
+.single-Product input {
+  width: 50px;
+  height: 40px;
+
+  margin-right: 10px;
+  padding-left: 10px;
+
+  font-size: 20px;
+
+  border: 1px solid #ff523b;
+}
+input:focus {
+  outline: none;
+}
+.single-Product .fa {
+  color: #ff523b;
+
+  margin-left: 10px;
+}
+.small-img-row {
+  display: flex;
+  justify-content: space-between;
+}
+.small-img-col {
+  flex-basis: 24%;
+
+  cursor: pointer;
+}
+.small-img-col img {
+  width: 100%;
+}
+
+/*----------Cart items------------*/
+
+.cart-page {
+  margin: 80px auto;
+}
+table {
+  width: 100%;
+
+  border-collapse: collapse;
+}
+.cart-info {
+  display: flex;
+  flex-wrap: wrap;
+}
+th {
+  padding: 5px;
+
+  color: #fff;
+  background-color: #ff523b;
+
+  text-align: left;
+
+  font-weight: normal;
+}
+td {
+  padding: 10px 5px;
+}
+td img {
+  width: 80px;
+  height: 80px;
+
+  margin-right: 10px;
+}
+td input {
+  width: 40px;
+  height: 30px;
+
+  padding: 5px;
+}
+td a {
+  color: #ff523b;
+
+  font-size: 12px;
+}
+.table-price {
+  display: flex;
+  justify-content: flex-end;
+}
+.table-price table {
+  width: 100%;
+  max-width: 350px;
+
+  border-top: 3px solid #ff523b;
+}
+td:last-child,
+th:last-child {
+  text-align: right;
+}
+
+/*-----------Account Page----------------*/
+
+.account-page {
+  padding: 50px 0;
+
+  background: radial-gradient(#fff, #ffd6d6);
+}
+.form-container {
+  width: 300px;
+  height: 400px;
+
+  position: relative;
+
+  margin: auto;
+  padding: 20px 0;
+  text-align: center;
+
+  background: #fff;
+  box-shadow: 0 0 20px 0 rgba(0, 0, 0, 0.1);
+
+  overflow: hidden;
+}
+.form-container span {
+  display: inline-block;
+  width: 100px;
+
+  padding: 0 10px;
+
+  font-weight: bold;
+
+  color: #555;
+
+  cursor: pointer;
+}
+.form-btn {
+  display: inline-block;
+}
+.form-container form {
+  position: absolute;
+  top: 130px;
+
+  max-width: 300px;
+
+  padding: 0 20px;
+
+  transition: transform 1s;
+}
+form input {
+  width: 100%;
+  height: 30px;
+
+  margin: 10px 0;
+  padding: 0 10px;
+
+  border: 1px solid #ccc;
+}
+form .btn {
+  width: 100%;
+
+  margin: 10px 0;
+
+  border: none;
+
+  cursor: pointer;
+}
+form .btn:focus {
+  outline: none;
+}
+#loginForm {
+  left: -300px;
+}
+#regForm {
+  left: 0;
+}
+form a {
+  font-size: 12px;
+}
+#indicator {
+  width: 100px;
+  height: 3px;
+
+  background: #ff523b;
+  border: none;
+
+  margin-top: 8px;
+
+  transform: translateX(100px);
+  transition: transform 1s;
+}
+
+/*-----------media query for menu-----------*/
+
+@media only screen and (max-width: 800px) {
+  nav ul {
+    position: absolute;
+    top: 70px;
+    left: 0;
+    width: 100%;
```

Comment:
if you want to group the properties then do it properly, in this file you have grouped different properties together which are not related to each other.. like in this case width  and position  are two different properties, don't group them together, and always try to keep width and height first while applying different properties.. @AbbasShaikh0304

Reply:
It happend only here, otherwise i've arrane them properly. i'll take account of your suggestion about width and height. Thanks @Tauqeer-Ahmed-99